packages/tools/pie-icons-webc/build.js

```diff
@@ -39,9 +39,9 @@ export class ${name} extends LitElement implements IconProps {
     // The following styles make sure that the icon will be sized correctly
     static styles = css\`
         :host svg {
-            display: var(--btn-icon-display, inline);
-            width: var(--btn-icon-size, 24px);
-            height: var(--btn-icon-size, 24px);
+            display: var(--icon-display-override);
```

Comment:
Could you also check that these changes work fine in modal (close btn) & icon button within nextjs Aperture please using Firefox and Safari. It might also be worth updating the iconButton.scss file with this new name too as it is using `--btn-icon-display`

Reply:
So far I tested storybook locally in Chrome, Firefox and Safari, all looks good. snapit build fails after [this change](https://github.com/justeattakeaway/pie/pull/1124) was merged so I can't check it in aperture

Reply:
Tested locally in pie aperture vanilla js and next js apps. Checked icon button, button, modal and cookie banner. I also created [a draft PR for pie-aperture](https://github.com/justeattakeaway/pie-aperture/pull/43) with dev package versions.